GOFevaluation/utils.py

```diff
@@ -352,32 +352,26 @@ def plot_equiprobable_histogram(data_sample, bin_edges, order=None,
         if nevents_expected is None:
             raise ValueError('nevents_expected cannot ' +
                              'be None while plot_mode=\'sigma_deviation\'')
-
         n_bins = get_n_bins(bin_edges)
         midpoint = nevents_expected / n_bins
-        delta = max(midpoint - ns.min(), ns.max() - midpoint)
-        sigma_deviation = delta / np.sqrt(midpoint)
         ns = (ns - midpoint) / np.sqrt(midpoint)
-        vmin = -sigma_deviation
-        vmax = sigma_deviation
```

Comment:
This makes things easier but I would like to revert this to the way it was implemented. By default, I think one should get the symmetric colorbar i.e. from `-sigma_deviation` to `sigma_deviation`. The only issue was that the arrows for out-of-bounds were misleading if the absolute values of vmin and vmax are identical.. Therefore I proposed  to replace `vmin` by `np.min(ns)` and `vmax` by `np.max(ns)`in the if-elif-elif structure to obtain `extend`. This is not the most beautiful solution unfortunately but should work fine.

Reply:
Shall I also give an error or warning when a user specifies unequal absolute values of `vmin` and `vmax` in `kwargs` for `sigma_deviation` mode?

Reply:
I just add a warning for users : )

Reply:
Very nice idea!!